mono/mini/intrinsics.c

```diff
@@ -166,6 +179,74 @@ llvm_emit_inst_for_method (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSign
 	return ins;
 }
 
+static MonoInst*
+emit_span_intrinsics (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsig, MonoInst **args)
+{
+	MonoInst *ins;
+
+	MonoClassField *ptr_field = mono_class_get_field_from_name (cmethod->klass, "_pointer");
+	if (!ptr_field)
+		/* Portable Span<T> */
+		return NULL;
+
+	if (!strcmp (cmethod->name, "get_Item")) {
```

Comment:
Don't we need to intrinsify set_Item as well?

Reply:
There is no set_Item, it returns a ref.

Reply:
So with Span ```
Span<int> span = ...;
span[0] = 10;
```

The store is compiled to this: ```
ldloc span
ldc.i4 10
call Span<int>::get_Item (int32)
stind.i4
```